app/scripts/modules/core/src/pipeline/config/templates/v2/pipelineTemplateV2.states.ts

```diff
@@ -0,0 +1,30 @@
+import { module } from 'angular';
+import { SETTINGS } from 'core/config';
+import { INestedState, STATE_CONFIG_PROVIDER, StateConfigProvider } from 'core/navigation/state.provider';
+import { PipelineTemplatesV2 } from './PipelineTemplatesV2';
+
+export const PIPELINE_TEMPLATES_V2_STATES_CONFIG = 'spinnaker.core.pipeline.templates.v2.states.config';
+
+module(PIPELINE_TEMPLATES_V2_STATES_CONFIG, [STATE_CONFIG_PROVIDER]).config([
+  'stateConfigProvider',
+  (stateConfigProvider: StateConfigProvider) => {
+    const pipelineTemplatesList: INestedState = {
+      name: 'mptv2',
+      url: '/mptv2',
```

Comment:
Would matching the url to the "pipeline-templates" name used elsewhere make sense here? I'm not sure if /mptv2 will be clear to users